packages/aws-cdk-lib/aws-events-targets/test/appsync/appsync.test.ts

```diff
@@ -0,0 +1,349 @@
+import * as path from 'path';
+import { Template } from '../../../assertions';
+import * as appsync from '../../../aws-appsync';
+import * as events from '../../../aws-events';
+import * as sqs from '../../../aws-sqs';
+import * as cdk from '../../../core';
+import * as targets from '../../lib';
+
+// GIVEN
+let stack: cdk.Stack;
```

Comment:
```suggestion
describe('AppSync GraphQL API target', () => {
  let stack: cdk.Stack;
  ...
```
Let's wrap it in a `describe` to keep it more organized.
Also, can you please remove the commented dead-code in the file?

Reply:
updated.